test/integrations/sources/shopify/data.ts

```
@@ -648,408 +668,410 @@ const serverSideEventsScenarios = [
-              token: '123456abcd',
-              total_discounts: '20.00',
-              total_discounts_set: {
-                shop_money: {
-                  amount: '20.00',
-                  currency_code: 'USD',
-                },
-                presentment_money: {
-                  amount: '20.00',
-                  currency_code: 'USD',
+              body: JSON.stringify({
+                id: 820982911946154508,
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 820982911946154500

(lint/correctness/noPrecisionLoss)